packages/e2e-test-app-fabric/test/__snapshots__/TextInputComponentTest.test.ts.snap

```diff
@@ -367,7 +367,7 @@ exports[`TextInput Tests Text have cursorColor 1`] = `
           "Brush Type": "ColorBrush",
           "Color": "rgba(0, 128, 0, 255)",
         },
-        "Offset": "83, 133, 0",
+        "Offset": "83, 5, 0",
```

Comment:
Any idea why the dramatic difference in Y coordinate due to this change? Give the TextInput test page a look just in case.

Reply:
All the other tests have a very small offset and this one was the only one with such a large y offset. So now the snapshot got updated and brought down the offset. Seems okay to me.